Fixed issue #11 and #12

llelectronics opened the pull request:
This should fix the issues with not resizing images. 
Tall images can be resized to desired size and swipe around to read everything clearly.

Commit:
Fixed issue #11 and #12

dicksonleong commented:
I can't test this since I don't have an actual device, and multitouch is not working on emulator. This seems to completely rewrite the ImagePage.qml so I reluctant to merge without proper testing myself.

llelectronics commented:
See this little demo video (audio might be a little bit low, sry for that): https://www.youtube.com/watch?v=lbSzWjGorJ4

I was not able to patch the original file in a matter that not only the PinchArea worked but also the Flickable would pickup the new hight of the image.
(To make the PinchArea work in your original file you have to move it down to the bottom making it basically the last object in the qml file. But that does only solve pinching. So you can pinch in but not move upwards or downwards.) 
But if you don't have a device to test it on it is really hard to fix this bug otherwise then.

dicksonleong commented:
While the pinch zooming is working it seems there are few problems: no pinch limit (notice the minScale and maxScale I have) and the width of the flickable also getting scaled (?) causes a lot of empty space on the left and right. Can you fix that?

llelectronics commented:
For the pinch limit I would only set a minScale as setting a maxScale does not make so much sense on very large images. 
I am not sure how to fix the width with the empty space yet. But I'll figure something out.

Commit:
Added video support

Commit:
Updated README.md and LICENSE file

Commit:
Fixed scaling of large images
Bumped version number

Commit:
9Gag now uses https only so switch to using it
Bumped rpm version

Commit:
Updated copyright year
Added changelog entry
Updated version in yaml file

Commit:
Changed REPO_WEBSITE url

Commit:
Updated unavailable channels and replaced with available ones

Commit:
Allow adding + removing sections in sectionPage
Bumped version number

llelectronics committed:
Comments API seems broken call mobile site instead
Added device Pixel Ratio Hacks for Webview

llelectronics committed:
Bump version

llelectronics committed:
Added devicePixelRatioHack
Commentspage updated (its not possible anymore to load comments without
official app)

llelectronics committed:
Disabled InfinityGag (not working anymore)
Updated changelog
Bumped version
